app/src/main/java/io/crate/node/CrateNode.java

```diff
@@ -54,16 +56,22 @@
         BlobPlugin.class,
         SrvPlugin.class,
         UDCPlugin.class,
+        URLRepositoryPlugin.class,
         S3RepositoryPlugin.class,
         Ec2DiscoveryPlugin.class,
+        CommonAnalysisPlugin.class,
         Netty4Plugin.class);
 
     protected CrateNode(Environment environment) {
         super(environment, CLASSPATH_PLUGINS);
     }
 
     @Override
-    protected void startUpLogging(Logger logger, Settings tmpSettings, boolean hadPredefinedNodeName) {
+    protected void startUpLogging(Logger logger,
+                                  Settings tmpSettings,
+                                  String nodeName,
+                                  String nodeId,
+                                  boolean hadPredefinedNodeName) {
```

Comment:
any reasons not to add the `nodeName/NodeId` logs ES is doing? see org.elasticsearch.node.Node L503

Reply:
Need to check that, I think we should keep the node name / node id logging.

Reply:
Included the nodeName/NodeID.